CHANGES.md

```diff
@@ -28,6 +28,13 @@
 
 OPCUA container image build with GitHub Actions has been fixed.
 
+### New features
+
+* base: Add motorPIGCS2 IOC and module, and image. by @guirodrigueslima in
+  https://github.com/cnpem/epics-in-docker/pull/64
+* base: Add scaler module. by @guirodrigueslima in 
+  https://github.com/cnpem/epics-in-docker/pull/64
```

Comment:
It looks like you've squashed the "base: add scaler module." commit in the rebase process.

Reply:
I've noticed now that both changes are also in `v0.8.1` instead of `Unreleased`.

Put them along with the pmac entry in the *end* of the New features section of the Unreleased version. This makes the changelog ordered by changes that were introduced in that specific release.